packages/design-system/src/components/Table/TableCell.jsx

```diff
@@ -69,9 +69,14 @@ export const TableCell = ({
       scope={defaultScope}
       headers={headers}
       id={id}
-      data-title={stackedTitle}
       {...tableCellProps}
     >
+      {/* Add column heading for stacked table */}
+      {_stackable && (
```

Comment:
The problem with this is that this requires the HTML only examples to mainly add this code, which is quite alot of boilerplate